Asterix/Example_param_file.ini

```diff
@@ -230,6 +230,17 @@ onbench=True
     # If monochromatic images (nb_wav = 1 or Delta_wav = 0), all these options are equivalent
```

Comment:
I can't comment above because it's too far from the code you touched:
The docstring in line 227 needs to be updated. It currently says that `nb_wav` is used for the estimation, which is not true anymore. Also, in line 228, the part: "This parameter is only for the estimation / correction." needs to be updated accordingly.

Reply:
I htink its correct. The iamges are still parametrized using the simulation wls defined in [modelconfig]

Reply:
Yes exactly, but not the estimation.